registry/mavrickrishi/modules/nexus/README.md

```diff
@@ -0,0 +1,153 @@
+---
+display_name: Sonatype Nexus Repository
+description: Configure package managers to use Sonatype Nexus Repository for Maven, npm, PyPI, and Docker registries.
+icon: ../../../../.icons/nexus.svg
+verified: true
+tags: [integration, nexus, maven, npm, pypi, docker]
+---
+
+# Sonatype Nexus Repository
+
+Configure package managers (Maven, npm, PyPI, Docker) to use Sonatype Nexus Repository with API token authentication.
```

Comment:
Can we also link to some SonaType Nexus docs on how to configure the package managers in Sonatype? An inline link would be sufficient.